pkg/mutation/assignmetadata.go

```diff
@@ -0,0 +1,72 @@
+package mutation
+
+import (
+	"fmt"
+
+	"github.com/open-policy-agent/gatekeeper/pkg/mutation/path/parser"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+// MetadataMutator is a mutator wrapping the AssignMetadata type.
+type AssignMetadataMutator struct {
+	//assignMetadata *mutationsv1.AssignMetadata
+	Path  parser.Path
```

Comment:
I think `Path` may be a function in the Mutator interface?

Reply:
Done